### PR TITLE
#162883623 Add user sign up with jwt token

### PR DESCRIPTION
**Pull request by GraceChiamaka**
### What will this PR do?
Create endpoints to signup a user

#### Description of Task to be completed?
Have endpoint '/api/v1/auth/signup' 

#### How should this be manually tested?
Using postman

#### Relevant pivotal tracker stories
[#162883623]

### CODE REVIEW

File: server/src/v1/controllers/meetup.js [hunk 1]
```diff
@@ -22,12 +22,10 @@ client.connect((err) => {
  * @param  {object} res
  */
 const createMeetup = (req, res) => {
-    const sqlQueryString = `INSERT INTO meetups (title, location, created_on, topic, images, tags, happening_on)
-                            VALUES ($1, $2, $3, $4, $5, $6, $7) RETURNING *`;
+    const sqlQueryString = `INSERT INTO meetups (location, topic, images, tags, happeningon)
```
Comment (tosmak16) [inline server/src/v1/controllers/meetup.js:25]: Well done @Amriesgrace Have you tried using `happeningOn`instead?

File: server/src/v1/controllers/users.js [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+    const sql = 'SELECT * FROM users where email = $1';
+    client.query(sql, reqEmail, (err, response) => {
+        if (response.rowCount === 0) {
+            return res.status(404).json({
+                status: 404,
+                message: 'There\'s no user with this email'
+            });
+        }
+        console.log(response.rows);
+        // console.log(response.rowCount);
```
Comment (tosmak16) [inline server/src/v1/controllers/users.js:104]: Well done @Amriesgrace Please remove console.log and the commented codes on line 103 and line 104 respectively.

File: server/src/v1/middleware/Auth.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    if (!bearerHeader) {
+        return res.status(401).json({
+            status: 401,
+            error: 'Auth Failed',
+        });
+    }
+    const tokenHeader = bearerHeader.split(' ');
+    const token = tokenHeader[1];
+
+    console.log('-----', token);
```
Comment (tosmak16) [inline server/src/v1/middleware/Auth.js:18]: @Amriesgrace Remove console log

File: server/src/v1/controllers/users.js (hunk 2 above)
Comment (GraceChiamaka) [reply server/src/v1/controllers/users.js:104]: Okay

File: server/src/v1/middleware/Auth.js (hunk 3 above)
Comment (GraceChiamaka) [reply server/src/v1/middleware/Auth.js:18]: Got It

File: server/src/v1/controllers/meetup.js (hunk 1 above)
Comment (GraceChiamaka) [reply server/src/v1/controllers/meetup.js:25]: Will do that

**Pull request merged**